src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,40 @@
-export const Tabs = () => {};
+import React from 'react';
+import classNames from 'classnames';
+import { Tab } from '../../type';
+
+type Props = {
+  tabs:Tab[]
+  isActiveId: string;
+  setActiveId:(tab: Tab) => void;
+
+};
+
```

Comment:
You need to add here (on tabs)  same const as on App: 

const activeTab = tabsArr.find(tab => tab.id === isActiveId) || tabsArr[0]; 

Because: 
Point 4: Pass the selectedTabId as a prop to the Tabs, the specified tab should be selected if possible (otherwise the first tab is selected).